ci/generate-snapshots/jobs/production-generate-snapshots.yml

```diff
@@ -12,8 +12,6 @@ jobs:
     trigger: false
   - get: aws-sdx
     trigger: false
-  - get: 4am-monday
```

Comment:
Why has this been removed? PR didn't mention it

Reply:
This was a temp last weekend to kick off the pipeline over the weekend so removed it at the same time. Should have mentioned on the PR.